homeassistant/bootstrap.py

```diff
@@ -261,25 +314,68 @@ def from_config_dict(config: Dict[str, Any],
             hass.config.config_dir = config_dir
             mount_local_lib_path(config_dir)
 
+    # async loop allready running
```

Comment:
I think that you should be able to do this regardless of if the loop is running or not:

```python
coro = async_from_config_dict(config, hass, config_dir, enable_log, verbose, skip_pip)
hass.loop.run_until_complete(future)
```